src/panels/MissionsPanel/MissionsPanel.tsx

```diff
@@ -23,5 +51,22 @@ export function MissionsPanel() {
     );
   }
 
-  return <MissionContent missionOverview={mission} />;
+  return (
+    <Box>
+      {data.missions.length > 1 && (
+        <Select
+          label={'Selected mission'}
+          placeholder={'Select a mission'}
+          data={data.missions.map((_mission) => {
+            return { value: _mission.identifier, label: _mission.name };
+          })}
```

Comment:
Personally I'm not a huge fan of the `_variableName` style. It looks to me like the member variables in OpenSpace, and for some reason I also find it hard to read... I would prefer a shorter name here like `entry`, `mis` , `element` or whatever